documentcloud/documents/tests/test_entity_extraction.py

```diff
@@ -0,0 +1,74 @@
+# Third Party
+import pytest
+
+# DocumentCloud
+from documentcloud.documents.entity_extraction import _get_or_create_entities
+
+# pylint: disable=too-many-lines, too-many-public-methods
+
+entity_dict = {
+    "name": "Knight",
+    "kind": 1,
+    "type_": 1,
+    "metadata": {"wikipedia_url": "https://en.wikipedia.org/wiki/Knight"},
+    "salience": 0.14932491,
+    "mentions": [
```

Comment:
Maybe less mentions are necessary for testing purposes?

Reply:
Ah, yes, that would be much more readable.